app/models/agents/rss_agent.rb

```diff
@@ -113,7 +113,7 @@ def check_urls(urls)
 
       created_event_count = 0
       sort_events(new_events).each.with_index do |event, index|
-        entry_id = event.payload[:id]
+        entry_id = event.payload[:update_id]
```

Comment:
Yep
On Jun 23, 2016 7:38 PM, "Andrew Cantino" <notifications@github.com> wrote:

> In app/models/agents/rss_agent.rb
> <https://github.com/cantino/huginn/pull/1553#discussion_r68346614>:
>
> > @@ -113,7 +113,7 @@ def check_urls(urls)
> >
> >        created_event_count = 0
> >        sort_events(new_events).each.with_index do |event, index|
> > -        entry_id = event.payload[:id]
> > +        entry_id = event.payload[:update_id]
>
> Will the re-fetch cause duplicate events to be emitted for users of
> RssAgents? If so, we should probably figure out how to migrate existing
> users.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cantino/huginn/pull/1553/files/936077e3c88fdde2393892effa6e39e67a858163#r68346614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADGq_nssGefDi4obVw7d4Af6e09i6uyVks5qO0MwgaJpZM4I5GJS>
> .
>